ruleset/rules/0755-office365_rules.xml

```diff
@@ -0,0 +1,928 @@
+        <group>SharePointContentTypeOperation</group>
+    </rule>
+
+    <rule id="91584" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^56$</field>
+        <description>Office 365: SharePoint list field events.</description>
+        <options>no_full_log</options>
+        <group>SharePointFieldOperation</group>
+    </rule>
+
+    <rule id="91585" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^57$</field>
+        <description>Office 365: Teams admin events.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftTeamsAdmin</group>
+    </rule>
+
+    <rule id="91586" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^58$</field>
+        <description>Office 365: Events related to HR data signals that support the Insider risk management solution.</description>
+        <options>no_full_log</options>
+        <group>HRSignal</group>
+    </rule>
+
+    <rule id="91587" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^59$</field>
+        <description>Office 365: Teams device events.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftTeamsDevice</group>
+    </rule>
+
+    <rule id="91588" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^60$</field>
+        <description>Office 365: Teams analytics events.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftTeamsAnalytics</group>
+    </rule>
+
+    <rule id="91589" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^61$</field>
+        <description>Office 365: Events related to compromised user alerts.</description>
+        <options>no_full_log</options>
+        <group>InformationWorkerProtection</group>
+    </rule>
+
+    <rule id="91590" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^62$</field>
+        <description>Office 365: Email campaign events from Microsoft Defender for Office 365.</description>
+        <options>no_full_log</options>
+        <group>Campaign</group>
+    </rule>
+
+    <rule id="91591" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^63$</field>
+        <description>Office 365: Endpoint DLP events.</description>
+        <options>no_full_log</options>
+        <group>DLPEndpoint</group>
+    </rule>
+
+    <rule id="91592" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^64$</field>
+        <description>Office 365: Automated incident response (AIR) events.</description>
+        <options>no_full_log</options>
+        <group>AirInvestigation</group>
+    </rule>
+
+    <rule id="91593" level="3">
```

Comment:
I think this level should be higher. Quarantine events seem to be critical.

Reply:
Done